src/progressive-profiling/ProgressiveProfiling.jsx

```diff
@@ -126,6 +136,7 @@ const ProgressiveProfiling = (props) => {
         isGenderSelected: !!values.gender,
         isYearOfBirthSelected: !!values.year_of_birth,
         isLevelOfEducationSelected: !!values.level_of_education,
+        variant: embeddedVariant,
```

Comment:
this will add `variant: ''` for the non-embedded experience. I think we don't want to add this property for the non-embedded experience.

Reply:
this concern is valid for other events as well.

Reply:
Does empty value affect our stats? I think it is fine if we are passing the '' value for non or ramp experience. We can easily differentiate the traffic on the welcome page by passing this property whatever the value would be.

Reply:
yes, you are right. but if we are never going to use this then why pass it?